app/controllers/catalog_controller.rb

```diff
@@ -1541,18 +1542,31 @@ def available_orchestration_templates
     available_orchestration_managers(@record.orchestration_template.id) if @record.orchestration_template
   end
 
-  def available_job_or_container_templates(manager_id)
+  def available_container_templates(manager_id)
     method = @edit[:new][:st_prov_type] == 'generic_ansible_tower' ? 'configuration_scripts' : 'container_templates'
     @edit[:new][:available_templates] =
       ExtManagementSystem.find_by(:id => manager_id).send(method).collect { |t| [t.name, t.id] }.sort
   end
 
+  def available_job_templates(manager_id)
+    @edit[:new][:available_templates] = []
+    all_job_templates = ExtManagementSystem.find_by(:id => manager_id).send('configuration_scripts').collect { |t| [t.name, t.id] }.sort
+    all_workflow_templates = ExtManagementSystem.find_by(:id => manager_id).send('configuration_workflows').collect { |t| [t.name, t.id] }.sort
```

Comment:
@h-kataria method `configuration_workflows` does not exist. You need to rework on this PR.

Reply:
@jameswnl please let me know what is the new method name that i should use. @bzwei i did test my PR as is yesterday and i was able to see correct values in the drop down.

Reply:
you can just filter what you've got from the previous query `all_job_templates` to get the workflows.

(If you are making another call to db to get just the `workflows`, that would defeat the purpose of the whole remodeling in `configuration_workflow` which is to allow ui to get back both types of objects in one db/api call)